### ISSUE TITLE
ch1.4训练一个分类器中可能存在多进程报错的问题

### ISSUE THREAD

**Issue by hebf181** [zergtant/pytorch-handbook#145]
使用的是macos系统。在运行过程中存在多进程报错的问题，应该在
trainloader = torch.utils.data.DataLoader(trainset, batch_size=4,
                                          shuffle=True, num_workers=0)
以及

testloader = torch.utils.data.DataLoader(testset, batch_size=4,
                                         shuffle=False, num_workers=0)
中设置进程数目为0
或者在训练的过程中使用if __name__=='__main__':将训练过程包起来。这样就可以正常运行不报错了。

多进程加载数据在windows系统中也不支持，在linux中可以。
转载自CSDN。
pytorch小白，欢迎交流
